File: Week 6 -List/Lab 6/Store/CashRegister.java
import java.text.*;

public class CashRegister {
    public double cash;

    public CashRegister() {
        cash = 0.0;
    }

    public void updateCash(double amount) {
        cash += amount;
    }

    @Override
    public String toString() {
        return "Cash: $" + formatted(cash);
    }

    private String formatted(double amount) {
        return new DecimalFormat("###,##0.00").format(amount);
    }
}

File: Week 6 -List/Lab 6/Store/CashRegister.py
class CashRegister:

    def __init__(self):
        self.cash = 0.0

    def update_cash(self, amount):
        self.cash = self.cash + amount
    
    def toString(self):
        amount='{:.2f}'.format(self.cash)
        if self.cash == 0.0:
            return "Cash register: empty"
        else:
            return "Cash: $" + str(amount)

